### PR TITLE
[amundsen-kube-helm] v1.1.0

### PR DESCRIPTION
**Pull request by javamonkey79**
### Summary of Changes

Fixes [344](https://github.com/lyft/amundsen/issues/344), [345](https://github.com/lyft/amundsen/issues/345)

### Documentation

Docs updated - check.

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by javamonkey79** [comment]
@lukelowery can you look this one over? Much smaller than the last one, and I think it will be useful for you too...

File: amundsen-kube-helm/templates/helm/templates/amundsen-service.yaml [hunk 1]
```diff
@@ -18,7 +18,6 @@ spec:
     - port: 5001
       name: {{ .Chart.Name }}-{{ .Values.search.serviceName }}-{{ .Values.environment }}-http
       targetPort: 5001
-      nodePort: 30001
```
Comment (javamonkey79) [inline amundsen-kube-helm/templates/helm/templates/amundsen-service.yaml]: I could not find a need to set the node port on a load balancer, but, if someone has a major objection, I can add it in with an optional lookup.

**Comment by javamonkey79** [comment]
Hey @feng-tao we have this running in prod, and it works fine. Can you merge, or, do you want to wait for some input? @lukelowery @bryfox perhaps?

**Comment by feng-tao** [comment]
it looks good. I thin it is good to merge.

**Comment by feng-tao** [comment]
thanks for the contribution!

**Comment by javamonkey79** [comment]
super, thanks @feng-tao !

**Pull request merged**